topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -93,7 +91,7 @@ private void createViews() {
 		// Views
 		logger.info("Create Views begins...");
 
-		loadWidgetsFromInsideOut();
+		CSS_BLOCK = loadWidgetsFromInsideOut();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)